core/examples/params.rs

```diff
@@ -0,0 +1,15 @@
+use jsonrpc_core::*;
+
+fn main() {
+	let mut io = IoHandler::new();
+
+	io.add_method("say_hello", |_params: Params| {
+    let parsed: Value = _params.parse().unwrap();
```

Comment:
Parsing to `Value` is not super useful, `.parse()` allow you to get anything that can be deserialized:
```
#[derive(Deserialize)]
struct HelloParams {
  name: String,
}
...

let parsed: HelloParams = params.parse().ok_or_else(Error::invalid_params)?;
Ok(Value::String(format!("{}", parsed.name));
```
Would be a more appropriate example imho.

Also:
1. Indentation is off (please reformat the entire file according to our `.editorconfig` rules).
2. `_params` should be renamed to `params` (the `_` in front indicates that parameter is unused)

Reply:
I see. Thank you for the review, and I will update it shortly.